index.html

```diff
@@ -1,158 +1,198 @@
+            <g>
+              <rect width="22.928" height="3.964"/>
+              <rect y="9.009" width="22.928" height="3.964"/>
+              <rect y="18.018" width="22.928" height="3.964"/>
+            </g>
           </svg>
+          <svg class="main-nav__svg main-nav__svg--close" version="1.0" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="22.199px" height="22.199px" viewBox="0 0 22.199 22.199" enable-background="new 0 0 22.199 22.199"
+            xml:space="preserve">
+            <polygon points="22.199,2.804 19.396,0 11.1,8.297 2.804,0 0,2.803 8.297,11.1 0,19.396 2.803,22.199 11.1,13.902 19.396,22.199 22.199,19.396 13.903,11.1 "/>
+          </svg>
+
         </div>
-      </header>
+        <a class="main-nav__link" href="index.html"></a>
+      </nav>
+
+      <div class="intro">
+        <h3 class="intro__header">Welcome to gorgeous Sedona. "Because the Grand Canyon sucks!"</h3>
+      </div>
+    </header>
+
+    <main class="main-content">
+      <section class="preamble">
+        <div class="preamble__inner">
+          <h3 class="preamble__header">SEDONA - a small town in the ARIZONA, deserves more!</h3>
+          <p class="preamble__paragraph">Consider 5 reasons why Sedona is cooler than Grand Canyon!</p>
+        </div>
+      </section>
 
-      <main class="main-content">
-        <section class="preamble">
-          <div class="preamble__inner">
-            <h3 class="preamble__header">SEDONA - a small town in the ARIZONA, deserves more!</h3>
-            <p class="preamble__paragraph">Consider 5 reasons why Sedona is cooler than Grand Canyon!</p>
-          </div>
-          <svg class="preamble__svg" xmlns="http://www.w3.org/2000/svg" version="1.1" width="100%" height="16" viewBox="0 0 100 5" preserveAspectRatio="none">
-                  <polygon points="0 0, 50 5, 100 0, 100 50, 0 50"></polygon>
+      <section class="sightseeing">
+        <div class="sightseeing__inner sightseeing__inner--top">
+          <svg class="sightseeing__svg" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+            width="320px" height="37px" viewBox="0 12.452 320 37" enable-background="new 0 12.452 320 37" xml:space="preserve">
+            <polygon points="320,12.452 0,12.452 0,34.175 160,49.452 320,34.175 "/>
           </svg>
-        </section>
+          <h3 class="sightseeing__header">The Present Town</h3>
+          <p class="sightseeing__counter-paragraph">— №1 —</p>
+          <p class="sightseeing__paragraph">Sedona is not a tourist attraction, the life flows in here on its own.</p>
+        </div>
+        <div class="image-wrapper">
+          <picture>
+            <source media="(min-width: 769px)" srcset="img/main-city-desktop@1x.jpg 1x, img/main-city-desktop@2x.jpg 2x">
+            <source media="(min-width: 480px)" srcset="img/main-city-tablet@1x.jpg 1x, img/main-city-tablet@2x.jpg 2x">
```

Comment:
Looking at here again I would still prefer to switch to tablet image at 320px, otherwise, it would look broken on many  devices  wider than 320

Reply:
we can look at it tomorrow